libs/scripts/scripts.js

```diff
@@ -21,103 +21,103 @@ const prodDomains = ['milo.adobe.com'];
+  ar: { ietf: 'es-AR', lang: 'es', reg: 'AR', tk: 'oln4yqj.css' },
+  br: { ietf: 'pt-BR', lang: 'pt', reg: 'BR', tk: 'inq1xob.css' },
+  ca: { ietf: 'en-CA', lang: 'en', reg: 'CA', tk: 'pps7abe.css' },
+  ca_fr: { ietf: 'fr-CA', lang: 'fr', reg: 'CA', tk: 'vrk5vyv.css' },
+  cl: { ietf: 'es-CL', lang: 'es', reg: 'CL', tk: 'oln4yqj.css' },
+  co: { ietf: 'es-CO', lang: 'es', reg: 'CO', tk: 'oln4yqj.css' },
+  la: { ietf: 'es-LA', lang: 'es', reg: '419', tk: 'oln4yqj.css' },
+  mx: { ietf: 'es-MX', lang: 'es', reg: 'MX', tk: 'oln4yqj.css' },
+  pe: { ietf: 'es-PE', lang: 'es', reg: 'PE', tk: 'oln4yqj.css' },
+  '': { ietf: 'en-US', lang: 'en', reg: 'US', tk: 'hah7vzn.css' },
   // EMEA
-  africa: { ietf: 'en', tk: 'pps7abe.css' },
-  be_fr: { ietf: 'fr-BE', tk: 'vrk5vyv.css' },
-  be_en: { ietf: 'en-BE', tk: 'pps7abe.css' },
-  be_nl: { ietf: 'nl-BE', tk: 'cya6bri.css' },
-  cy_en: { ietf: 'en-CY', tk: 'pps7abe.css' },
-  dk: { ietf: 'da-DK', tk: 'aaz7dvd.css' },
-  de: { ietf: 'de-DE', tk: 'vin7zsi.css' },
-  ee: { ietf: 'et-EE', tk: 'aaz7dvd.css' },
-  es: { ietf: 'es-ES', tk: 'oln4yqj.css' },
-  fr: { ietf: 'fr-FR', tk: 'vrk5vyv.css' },
-  gr_en: { ietf: 'en-GR', tk: 'pps7abe.css' },
-  ie: { ietf: 'en-GB', tk: 'pps7abe.css' },
-  il_en: { ietf: 'en-IL', tk: 'pps7abe.css' },
-  it: { ietf: 'it-IT', tk: 'bbf5pok.css' },
-  lv: { ietf: 'lv-LV', tk: 'aaz7dvd.css' },
-  lt: { ietf: 'lt-LT', tk: 'aaz7dvd.css' },
-  lu_de: { ietf: 'de-LU', tk: 'vin7zsi.css' },
-  lu_en: { ietf: 'en-LU', tk: 'pps7abe.css' },
-  lu_fr: { ietf: 'fr-LU', tk: 'vrk5vyv.css' },
-  hu: { ietf: 'hu-HU', tk: 'aaz7dvd.css' },
-  mt: { ietf: 'en-MT', tk: 'pps7abe.css' },
-  mena_en: { ietf: 'en', tk: 'pps7abe.css' },
-  nl: { ietf: 'nl-NL', tk: 'cya6bri.css' },
-  no: { ietf: 'no-NO', tk: 'aaz7dvd.css' },
-  pl: { ietf: 'pl-PL', tk: 'aaz7dvd.css' },
-  pt: { ietf: 'pt-PT', tk: 'inq1xob.css' },
-  ro: { ietf: 'ro-RO', tk: 'aaz7dvd.css' },
-  sa_en: { ietf: 'en', tk: 'pps7abe.css' },
-  ch_de: { ietf: 'de-CH', tk: 'vin7zsi.css' },
-  si: { ietf: 'sl-SI', tk: 'aaz7dvd.css' },
-  sk: { ietf: 'sk-SK', tk: 'aaz7dvd.css' },
-  ch_fr: { ietf: 'fr-CH', tk: 'vrk5vyv.css' },
-  fi: { ietf: 'fi-FI', tk: 'aaz7dvd.css' },
-  se: { ietf: 'sv-SE', tk: 'fpk1pcd.css' },
-  ch_it: { ietf: 'it-CH', tk: 'bbf5pok.css' },
-  tr: { ietf: 'tr-TR', tk: 'aaz7dvd.css' },
-  ae_en: { ietf: 'en', tk: 'pps7abe.css' },
-  uk: { ietf: 'en-GB', tk: 'pps7abe.css' },
-  at: { ietf: 'de-AT', tk: 'vin7zsi.css' },
-  cz: { ietf: 'cs-CZ', tk: 'aaz7dvd.css' },
-  bg: { ietf: 'bg-BG', tk: 'aaz7dvd.css' },
-  ru: { ietf: 'ru-RU', tk: 'aaz7dvd.css' },
-  ua: { ietf: 'uk-UA', tk: 'aaz7dvd.css' },
-  il_he: { ietf: 'he', tk: 'nwq1mna.css', dir: 'rtl' },
-  ae_ar: { ietf: 'ar', tk: 'nwq1mna.css', dir: 'rtl' },
-  mena_ar: { ietf: 'ar', tk: 'dis2dpj.css', dir: 'rtl' },
-  sa_ar: { ietf: 'ar', tk: 'nwq1mna.css', dir: 'rtl' },
+  africa: { ietf: 'en', lang: 'en', reg: '002', tk: 'pps7abe.css' },
```

Comment:
I was just curious where you got the '002' value, I don't see a mention of it on https://developers.google.com/recaptcha/docs/language

Reply:
`002` is the [UN M49](https://en.wikipedia.org/wiki/UN_M49) code for Africa. You can use UN M49 for region subtag [BCP 47](https://en.wikipedia.org/wiki/IETF_language_tag).